media/pages/sk/podporte-nas/page.html

```diff
@@ -38,37 +26,35 @@ <h3>2% z daní</h3>
-<em>* Podmienka darovania 2% je, že sumu vo výške minimálne 0,5% dane daruje právnická osoba organizácii nezriadenej za účelom podnikania. Nemusí to byť nevyhnutne ten istý subjekt, ktorému PO asignuje 2% dane.</em>
+        <em>* Podmienka darovania 2% je, že sumu vo výške minimálne 0,5% dane daruje právnická osoba organizácii nezriadenej za účelom podnikania. Nemusí to byť nevyhnutne ten istý subjekt, ktorému PO asignuje 2% dane.</em>
       </li>
       <li>
         <i class="chv-icon chv-icon-li chv-icon-lg icon-ok chv-color-blue-ll"></i>
         Ako si ako <strong>fyzická osoba</strong> podávate daňové priznanie sami, vypočítajte si 2% zo zaplatenej dane a jednoducho vpíšte nasledovné údaje do Vášho daňového priznania:<br/>
-<strong>IČO:</strong> <em>(4 políčka nechajte voľné)</em> 50436376<br/>
-<strong>Obchodné meno alebo názov:</strong> Chcemvediet.sk<br/>
-<strong>Ulica:</strong> Karpatské námestie <strong>Súpisné/orientačné číslo:</strong>10A<br/>
-<strong>PSČ:</strong>83106 <strong>Obec</strong>Bratislava
-<br/>
-<em>* Ak ste neoznámili daňovému úradu predĺženie termínu, posledný termín na jeho podanie je 31.3.2018</em>
+        <strong>IČO:</strong> <em>(4 políčka nechajte voľné)</em> 50436376<br/>
+        <strong>Obchodné meno alebo názov:</strong> Chcemvediet.sk<br/>
+        <strong>Ulica:</strong> Karpatské námestie <br/>
+        <strong>Súpisné/orientačné číslo:</strong>10A<br/>
+        <strong>PSČ:</strong>83106<br/>
+        <strong>Obec:</strong> Bratislava<br/>
+        <em>* Ak ste neoznámili daňovému úradu predĺženie termínu, posledný termín na jeho podanie je 31.3.2018</em>
       </li>
-
-     <li>
+      <li>
         <i class="chv-icon chv-icon-li chv-icon-lg icon-ok chv-color-blue-ll"></i>
         Ak ste <strong>zamestnanec</strong>, požiadajte do 15. 2. 2018 zamestnávateľa o vykonanie ročného zúčtovania zaplatených preddavkov na daň. Potom požiadajte zamestnávateľa o vystavenie tlačiva <em>Potvrdenie o zaplatení dane</em>. Z tohto potvrdenia si viete zistiť dátum zaplatenia dane a vypočítať 2% z Vašej zaplatenej dane (to musí byť minimálne 3 €). Potrebné údaje vpíšte do tlačiva <a href="/sk/podporte-nas/zamestnanci-chcemvediet.pdf">Vyhlásenie o poukázaní 2%</a>. Najneskôr do 30. apríla 2018 pošlite/doručte formuláre daňovému úradu v mieste vášho bydliska.
+      </li>
     </ul>
-   </div>
+  </div>
 </div>
 
 <h2>Aké sú naše výsledky?</h2>
 
 <div class="row">
   <div class="col-sm-6">
-
-Od roku 2016 je portál Chcemvediet.sk prostriedkom k tomu, aby bežní ľudia mohli pohodlne podávať infožiadosti. Medzi hlavné metriky, ktoré sledujeme, patrí počet <strong>registrovaných užívateľov</strong>, <strong>podaných infožiadostí</strong> a počet <strong>inštitúcií v databáze</strong>.
-
-Ak vás zaujíma činnosť OZ Chcemvediet.sk podrobnejšie, bližšie informácie nájdete v našich výročných správach: 
-
```

Comment:
@mmmaly Pls vazne, dodrziavaj odsadzovania a nevynechavaj uzatvaracie tagy (tie vynechavas preto, ze neodsadzujes a potom si nevsimnes). Kto ma to potom upratovat?